Cargo.toml

```diff
@@ -5,3 +5,11 @@ edition = "2018"
 
 [dependencies]
 anyhow = "1.0.42"
+ed25519-dalek = "1.0.1"
+signature = "1.3.1"
```

Comment:
What is this dependency for?

Reply:
Signature is a crate that contains a trait needed  in scope to work with ed25519_dalek signatures